wavmio/stub.go

```diff
@@ -133,12 +133,12 @@ func AdvanceInboxMessage() {
 	seqAdvanced++
 }
 
-func ResolvePreImage(hash common.Hash) []byte {
+func ResolvePreImage(hash common.Hash) ([]byte, error) {
 	val, ok := preimages[hash]
 	if !ok {
 		panic("preimage not found")
```

Comment:
Now you can return an error here

```suggestion
		return nil, errors.New("preimage not found")
```

Reply:
I think we need to panic since it's in the replay binary, and it should never happen.

Reply:
It might make sense to return an error but have it panic at the call site. You're already panicking where you need to should its callers receive an error